docs/csharp/fundamentals/functional/discards.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Discards - C# Guide
+title: Discards - unassigned discardable variables
 description: Describes C#'s support for discards, which are unassigned, discardable variables, and the ways in which discards can be used.
 ms.technology: csharp-fundamentals
-ms.date: 09/22/2020
+ms.date: 05/14/2021
 ---
-# Discards - C# Guide
+# Discards - C# Fundamentals
```

Comment:
Some of the other articles aren't using C# Fundamentals, and even removed C# Programming Guide. Why is this one using something in the H1?

Reply:
I'd like to handle this in a follow-up PR. I started editing this thinking I could avoid duplication, but I ran into several locations where the H1 or title matched another article in the docset. (This is one, where there is a similar article in language reference).

What I'd like to do as I edit and extend this area is this:

1. Add a title suffix for "C# Fundamentals" for all articles under this node.
1. Make sure the H1s are distinct from the title, and distinct in the docset.

Does that make sense? If so, I'll make a new issue for that plan.